blazor/diagram/connectors/segments/bezier/bezier.md

```diff
@@ -242,4 +242,109 @@ Also, if you provide segments during the initial rendering, the segment collecti
 ```
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/Connectors/Segments).
 
-![Avoid overlapping with bezier](../../../images/bezierOverlap.png)
\ No newline at end of file
+![Avoid overlapping with bezier](../../../images/bezierOverlap.png)
+
+### How to customize Bezier Segment Thumb Shape 
+
+The bezier connector can have any number of segments in between the source and the target point. By default, these segments are rendered as circles . The `Shape` property of `SegmentThumbSettings` allows to define the shape of the segment thumb.
```

Comment:
The Bezier connector can have multiple segments between the source and target points. By default, these segments are rendered as circles, but this can be customized either globally or for individual connectors using the **SegmentThumbSettings** class.

To change the segment thumb shape for all Bezier connectors, configure the **SegmentThumbSettings** property of the SfDiagramComponent class and set the **Shape** property to the desired shape.

To customize the segment thumb shape for a specific connector, first disable the **InheritSegmentThumbShape** constraint. Then, configure the **SegmentThumbSettings** property of the Connector class, specifying the desired shape using the **Shape** property of the **SegmentThumbSettings** class.

Reply:
Add API links for bold text